### DISCUSSION TITLE
Feature request: `backward-kill-word/eof`

### DISCUSSION THREAD

**Discussion by yrammos** [junegunn/fzf#3304] [Ideas]
`backward-delete-char/eof` is a remarkably useful feature — one naturally aborts fzf upon deleting the last char of a query.

Could the same be implemented for kill-word actions that delete the last word of a query (something like `backward-kill-word/eof`)? 

Thanks for your consideration.

**Comment by junegunn** [comment]
Have you tried this?

> backward-eof
>        Triggered when the query string is already empty and you try to delete it backward.
>
>        e.g.
>             fzf --bind backward-eof:abort

**Reply by yrammos** [reply]
Perfect — thank you!
